apps/bfd-pipeline/bfd-pipeline-rda-grpc/src/main/java/gov/cms/bfd/pipeline/rda/grpc/source/McsClaimTransformer.java

```diff
@@ -1042,7 +1042,7 @@ private PreAdjMcsLocation transformMessageImpl(
     transformer.copyEnumAsString(
         namePrefix + PreAdjMcsLocation.Fields.idrLocActvCode,
         true,
-        1,
+        0,
```

Comment:
I'm not totally sure about what this number is, but based on how many were updated, should it be a constant value?

Reply:
Previously the `minLength` defaulted to 1.  Now most enums need to use `0` instead.  But that's a good point.  I'll make the default `0` and just pass in `1` in the few places that need it.

Reply:
That's the minimum allowed string length.

Reply:
I have refactored the code to eliminate the explicit zeros.